src/chttpd/src/chttpd_sup.erl

```diff
@@ -41,8 +49,8 @@ init([]) ->
         ?CHILD(chttpd, worker),
         ?CHILD(chttpd_auth_cache, worker),
         {chttpd_auth_cache_lru,
-	 {ets_lru, start_link, [chttpd_auth_cache_lru, lru_opts()]},
-	 permanent, 5000, worker, [ets_lru]}
+            {ets_lru, start_link, [chttpd_auth_cache_lru, lru_opts()]},
```

Comment:
Lets be cleaner here and undo this hunk.

Reply:
Fixed it